CHANGELOG.md

```diff
@@ -17,14 +17,15 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
-## [Unreleased]
-### Fixed
-- Fixes for type annotations for Functions API
+## [7.75.0] - 2025-04-22
 ### Added
+- Support for data modeling query set operations - union, unionAll, and intersection
 - Support for the `/simulators/logs` API endpoint.
 - Support for the `/simulators/routines` and `/simulators/routines/revisions` API endpoints.
 - Support for the `/simulators/models` and `/simulators/models/revisions` API endpoints.
 - Support for the `/simulators` and `/simulators/integration` API endpoints.
+### Fixed
```

Comment:
Let’s keep simulator-stuff in the «unreleased» section until they have added all the APIs and want to «make it official»!

…unless that it now and I’m just uninformed 😅

Lastly, do we need that change log entry regarding type annotations? 🤔

Reply:
AFAICT the APIs are there and can be used, so they should be part of the changelog IMO.